src/Calculator/App.xaml

```diff
@@ -1609,6 +1619,97 @@
                 </Setter>
             </Style>
 
+            <Style x:Name="EquationButtonStyle" TargetType="Button">
+                <Setter Property="Template">
+                    <Setter.Value>
+                        <ControlTemplate TargetType="Button">
+                            <ContentPresenter x:Name="ContentPresenter"
+                                              Padding="{TemplateBinding Padding}"
+                                              HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                              VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                              Background="{TemplateBinding Background}"
+                                              BorderBrush="{TemplateBinding BorderBrush}"
+                                              BorderThickness="{TemplateBinding BorderThickness}"
+                                              AutomationProperties.AccessibilityView="Raw"
+                                              BackgroundSizing="{TemplateBinding BackgroundSizing}"
+                                              Content="{TemplateBinding Content}"
+                                              ContentTemplate="{TemplateBinding ContentTemplate}"
+                                              ContentTransitions="{TemplateBinding ContentTransitions}"
+                                              CornerRadius="{TemplateBinding CornerRadius}">
+
+                                <VisualStateManager.VisualStateGroups>
+                                    <VisualStateGroup x:Name="CommonStates">
+                                        <VisualState x:Name="Normal">
+
+                                            <Storyboard>
+                                                <PointerUpThemeAnimation Storyboard.TargetName="ContentPresenter"/>
+                                            </Storyboard>
+                                        </VisualState>
+
+                                        <VisualState x:Name="PointerOver">
+
+                                            <Storyboard>
```

Comment:
You can fix the glitchy icon we were seeing by deleting this StoryBoard and replacing with this:

`                                            <VisualState.Setters>
                                                <Setter Target="ContentPresenter.(ContentPresenter.Content)" Value="&#xE890;"/>
                                                <Setter Target="ContentPresenter.(ContentPresenter.FontFamily)" Value="{ThemeResource SymbolThemeFontFamily}"/>
                                            </VisualState.Setters>`

You will need to also add in the other background changes, etc.